Delejez/Bosszúcska

```diff
@@ -0,0 +1,68 @@
+Italt mérgezni, csapba vérezni
+Lánccal folytani, bűzzel gyilkolni
+Anyád kurvája
+Csapba vérezni, italt mérgezni
+Lángok közt édes Klementin
+
+Dokit leszarni, csapba vérezni
+Áh ezaz *kacsint*, rajta túlleni
+Anyád kurvája
+Áh ezaz *kacsint*, dokit megölni, felégni
+
+Oh megtorlásom
+Része leszel a vetetésért
+Csak ki kell főzni, bébi
+Ah-ah-ah, ah-ah-ah-ah
+
+Megtorlásom
+Része leszel a vetetésért
+Csak ki kell főzni, bébi
+Ah-ah-ah, ah-ah-ah-ah
+
+Bűzzel gyilkolni, csapba vérezni
+Italt mérgezni, ezen túlleni
+Anyád kurvája
+Csapba vérezni, férjed leszarni
+Lángok közt drága Klementin
+
+Italt mérgezni, csapba vérezni
+Lánccal folytani, bűzzel gyilkolni
+Anyád kurvája
+Csapba vérezni, italt mérgezni, felégni
+
+Megtorlásom
+Része leszel a vetetésért
+Csak ki kell főzni, bébi
+Ah-ah-ah, ah-ah-ah-ah
+
+Megtorlásom
+Része leszel a vetetésért
+Csak ki kell főzni, bébi
+Ah-ah-ah, ah-ah-ah-ah
+
+Felnevetett
+S azt mondta
+"Menj innen"
+
+Felkacagott
+S azt mondta, azt mondta
+"Menj innen, el innen"
+
+Ah-ah
+Ah-ah
+
+Megtorlásom
+Része leszel a vetetésért
+Csak ki kell főzni, bébi
+Ah-ah-ah, ah-ah-ah-ah
+
+Megtorlásom
+Része leszel a vetetésért
+Csak ki kell vinni, főzni, sütni, lopni, csalni
+Ah-ah-ah, ah-ah
+
+Ho
+Drága Klementin
+Ho
+Élned, lenned kellet volna ma
+Élned, lenned kellet volna ma
```

Comment:
Shoulda been, coulda been, woulda been, woulda been you
Lehetnél, lehettél, te lettél, lehetnél te

Reply:
Ez way better